venti/rootfs/cleanup.sh

```diff
@@ -2,8 +2,6 @@
 
 set -e
 
-localepurge
```

Comment:
Why is this (and similar lines in other files) removed?  Is it no longer needed (e.g. because it is run automatically)?

Reply:
Current images don't contain any locales except these two we are generating:
```
+ chroot /rootfs/ /usr/sbin/locale-gen
Generating locales (this might take a while)...
  en_US.UTF-8... done
  en_US.ISO-8859-1... done
Generation complete.
```